src/simplewallet/simplewallet.h

```diff
@@ -49,6 +49,8 @@
 
 #undef MONERO_DEFAULT_LOG_CATEGORY
 #define MONERO_DEFAULT_LOG_CATEGORY "wallet.simplewallet"
+// Hardcode Monero's donation address (see #1447)
+#define MONERO_DONATION_ADDR "44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A"
```

Comment:
Can we get in the habit of using `constexpr const char MONOERO_DONTATION_ADDR[]` for this? No need to have a macro expansion on this token. Although it is following the "style" of this file.

Reply:
Agreed.

Reply:
What are the benefits ?

Reply:
One is specifying a string constant, the other is specifying a token to be expanded before parsing the language. Since macro expansion is never needed, its the wrong tool for the job. The only negative I can think of doesn't happen in this situation - getting the address of the macro expansion will behave differently than getting the address of a string constant.

Reply:
I see no reason to do this unless you know it's a frequently used constant, and thus interning it consolidates multiple uses into a single memory location. Otherwise, #define'd constants are a common idiom already and there's no point in quibbling over this.

Reply:
>common idiom

A C hangover, for sure.

>quibbling

But it's [too much fun](https://web.archive.org/web/20170724031517/http://www.gotw.ca/gotw/032.htm) with hyc ;)